src/default-permalink-content-renderer.js

```diff
@@ -6,9 +6,10 @@ var styles = require('css!./css/styles.css');
 var sdkStyles = require('css!streamhub-sdk/css/style.css');
 var packageAttribute = require('./package-attribute');
 
-var defaultPermalinkContentHandler = function (content) {
+var defaultPermalinkContentHandler = function (content, opts) {
+    opts = opts || {};
     //Get the view for the content
-    var cvf = permalinkViewFactory(),
```

Comment:
Passing it into `permalinkViewFactory` will mixin [some behavior](https://github.com/Lizards/streamhub-permalink/blob/support_contentViewFactory_param/src/permalink-view-factory.js#L26) that you (or someone else using this) may not necessarily want if they're trying to pass a completely custom contentViewFactory.

How about this line as
```javascript
var cvs = opts.contentViewFactory || permalinkViewFactory();
```